bus-mapping/src/evm/opcodes/error_oog_codestore.rs

```diff
@@ -0,0 +1,49 @@
+use crate::{
+    circuit_input_builder::{CircuitInputStateRef, ExecStep},
+    evm::Opcode,
+    operation::CallContextField,
+    Error,
+};
+use eth_types::GethExecStep;
+
+#[derive(Debug, Copy, Clone)]
+pub struct ErrorOOGCodeStore;
+
+impl Opcode for ErrorOOGCodeStore {
+    fn gen_associated_ops(
+        state: &mut CircuitInputStateRef,
+        geth_steps: &[GethExecStep],
+    ) -> Result<Vec<ExecStep>, Error> {
+        let geth_step = &geth_steps[0];
+        let mut exec_step = state.new_step(geth_step)?;
+        let next_step = if geth_steps.len() > 1 {
+            Some(&geth_steps[1])
+        } else {
+            None
+        };
+        exec_step.error = state.get_step_err(geth_step, next_step).unwrap();
```

Comment:
let next_step = geth_steps.get(1).  ?

Reply:
updated in [fbea334](https://github.com/scroll-tech/zkevm-circuits/pull/304/commits/fbea3347b5e799723a89c9a3c8e2b5eb05579c22) thanks